components/list/list-item.js

```diff
@@ -169,6 +218,9 @@ class ListItem extends RtlMixin(LitElement) {
 			}
 		});
 	}
+	_handleChange(e) {
+		this.checked = e.target.checked;
```

Comment:
We probably want to fire an event, something like `d2l-list-item-selection-change`?

Reply:
See PR 4.

This is only the rendering logic.